### PR TITLE
chore: bump deckgl and mapbox libs

### PR DESCRIPTION
**Pull request by villebro**
🐛 Bug Fix

The deckgl plugin started crashing my browser, and bumping both the `deck.gl` and `mapbox-gl` packages to the latest stable versions seemed to fix the problem (currently on 2 years old versions).

### AFTER
image

### BEFORE
image

### CODE REVIEW

**Comment by kgabryje** [comment]
It's a pretty big bump. Are there no breaking changes?

File: packages/superset-ui-legacy-preset-chart-deckgl/package.json [hunk 1]
```diff
@@ -30,24 +30,24 @@
...
     "urijs": "^1.18.10",
     "xss": "^1.0.6"
   },
   "peerDependencies": {
-    "@superset-ui/chart-controls": "^0.17.12",
-    "@superset-ui/core": "^0.17.11",
-    "react": "^15 || ^16"
+    "@superset-ui/chart-controls": "^0.17.80",
+    "@superset-ui/core": "^0.17.80",
+    "react": "^16.13.1"
```
Comment (villebro) [inline packages/superset-ui-legacy-preset-chart-deckgl/package.json:51]: `react-map-gl` requires `>=16.3`. Also, this is what `superset-ui` requires.

**Review by zhaoyongjie — APPROVED** [review]
LGTM

**Comment by villebro** [comment]
> It's a pretty big bump. Are there no breaking changes?

This is the biggest change that we need to be wary of (bumping to 1.0 means opting in to the new pricing model): https://blog.mapbox.com/new-pricing-46b7c26166e7

**Review by kgabryje — APPROVED** [review]
LGTM!

**Pull request merged**